### ISSUE TITLE
Events should not be part of the resources in cluster mode

### ISSUE THREAD

**Issue by chargio** [kubeshop/monokle#3637]
Events are not configuration resources, and they can be a very high number.
Events should be present in their own interface, not mixed with other resources.

Delete events from the list of resources

**Comment by devcatalin** [comment]
For this one we should filter the events from the UI, but I think we still need to have them in memory @erdkse ?

**Comment by erdkse** [comment]
yes, We should listen to the Events for the new ones at least, or we can completely remove them from our kindHandlers and create a separate listener for the events.

We should consider this [issue](https://github.com/kubeshop/monokle/issues/3657) together I believe. 

I also created a [Draft PR](https://github.com/kubeshop/monokle/pull/3670) that just removes Events from the left menu.

**Comment by olensmar** [comment]
My understanding is that clicking an event in the dashboard goes to the list of events - so if we remove this we need to provide some alternative target!?
